flexmeasures/auth/tests/test_perm.py

```diff
@@ -97,61 +103,59 @@ def test_consultant_account_update_perm(
 
 
 @pytest.mark.parametrize(
-    "requesting_user, account_name, has_perm",
+    "requesting_user, required_perm, has_perm",
     [
-        ("test_consultant@seita.nl", "Test ConsultancyClient Account", True),
-        ("test_consultant@seita.nl", "Test Supplier Account", False),
+        ("test_consultant@seita.nl", "delete", True),
```

Comment:
I believe three test cases would be enough as you test each permission to work on client and not work on non-client.

Or you include client and non-client account names in the parameter set. Like in `test_consultant_can_work_on_clients_asset()` - might be good if they are aligned.

As it is now, it is hard to read these parameter sets and know what you are testing.